lib/matplotlib/quiver.py

```diff
@@ -239,6 +174,52 @@ def __init__(self, Q, X, Y, U, label,
                  *, angle=0, coordinates='axes', color=None, labelsep=0.1,
                  labelpos='N', labelcolor=None, fontproperties=None,
                  **kw):
+        """
+        Add a key to a quiver plot.
+
+        Parameters
+        ----------
+        Q : `Quiver`
+        X, Y: scalar
+            The location of the key.
+
+            The positioning of the key depends on *X*, *Y*, *coordinates*,
```

Comment:
I would put this section above the Parameters section as it describes the positioning more generally and does also explain labelpos and coordinates.

Reply:
sure, changed